default/python/AI/ShipDesignAI.py

```diff
@@ -1760,7 +1740,7 @@ def _starting_guess(self, available_parts, num_slots):
 
     def _calc_rating_for_name(self):
         self.update_stats(ignore_species=True)
-        return self.design_stats.structure*self._total_dmg()*(1+self.design_stats.shields/10)
+        return self.design_stats.structure*self._total_dmg()*(1+self.design_stats.shields//10)
```

Comment:
should be true division.

Reply:
My goal was to make code work same as it.  Lets change it to true division.